src/NuGet.Core/NuGet.Protocol/Resources/RawSearchResourceV3.cs

```diff
@@ -36,7 +37,14 @@ public RawSearchResourceV3(HttpSource client, IEnumerable<Uri> searchEndpoints)
             _searchEndpoints = searchEndpoints.ToArray();
         }
 
-        public virtual async Task<JObject> SearchPage(string searchTerm, SearchFilter filters, int skip, int take, Common.ILogger log, CancellationToken cancellationToken)
```

Comment:
Maybe you discussed this in earlier iterations, but isn't this a binary breaking change? 

cc @zivkan

Reply:
I doubt it breaks anyone but just ensuring we're aware of what we're doing here.

Reply:
Good catch, thanks Nikolche. I was so focused on checking new APIs, I forgot to check for breaking APIs.

I agree that people shouldn't be using this directly, they should be using `SourceRepository.GetResource<PackageSearchResource>()`. Still, we shouldn't break APIs.

Reply:
@zivkan  @nkolev92 what do I do here? Put it back along the new API.

Reply:
@zivkan @nkolev92 I put back old API and marked as Obsolete so it wouldn't break other applications that depend on old json API.